task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -48,11 +54,12 @@ class KubernetesPodExecutor(TaskExecutor):
     def __init__(self, namespace: str) -> None:
         self.kube_client = KubeClient()
         self.namespace = namespace
-
+        ensure_namespace(self.kube_client, namespace)
```

Comment:
do we actually want this? i'd rather have this machinery fail if namespace is missing as that would indicate critical misconfiguration on another level.

Reply:
we were just copying this pattern from paasta - but I guess maybe we should just create the namespace in puppet and assume that that's been applied anywhere where we're running tron?

Reply:
Removed 👍